nf_core/pipeline-template/README.md

```diff
@@ -73,6 +73,19 @@ The results obtained from the full-sized test can be viewed on the [nf-core webs
 
 {% if branded -%}
 
+## Codespaces
```

Comment:
Can we add this to [`CONTRIBUTING.md`](https://github.com/nf-core/tools/blob/dev/nf_core/pipeline-template/.github/CONTRIBUTING.md) instead please?

We should move this file to the root directory and mention it the readme to make it more obvious. It is mentioned further down in the readme but I don't think it's obvious enough currently.
That file also needs updating and some additions. Can take this in a separate issue / PR though.

Reply:
Great idea.